MAUI/ImageEditor/toolbar-customization.md

```diff
@@ -0,0 +1,530 @@
+---
+layout: post
+title: Toolbar Customization in .NET MAUI Image Editor Control | Syncfusion
+description: Learn here all about toolbar customization support in the Syncfusion .NET MAUI ImageEditor (SfImageEditor) control.
+platform: .NET MAUI
+control: SfImageEditor
+documentation: ug
+---
+
+# Toolbar Customization in .NET MAUI Image Editor (SfImageEditor)
+
+You can customize the color palette, visibility, and appearance of each toolbar item.
+
+## Customize toolbar items
+
+The image editor control provides support to customize and configure the appearance of toolbar menu. You can customize the toolbar by adding respective FooterToolbarItem and HeaderToolbarItem.
+
+### Toolbar item
+
+You can customize each toolbar item using the `Name` and `Icon` properties.
+
+### Name
+
+You can get or set the names of built-in toolbar and dynamically added toolbar items using the `Name` property.
+
+{% highlight C# %}
+
+            public MainPage()
+            {			
+                InitializeComponent();
+                SfImageEditor imageEditor = new SfImageEditor();
+                imageEditor.Source = ImageSource.FromFile("image4.png");
+                string itemName = imageEditor.Toolbars[1].ToolbarItems[0].Name;
+            }
+
+{% endhighlight %}
+
+N> The following built-in toolbar item names are available in image editor: `Browse`, `Separator`, `Undo`, `Redo`, `ZoomOut`, `ZoomIn`, `Reset`, `Save`, `SaveEdit`, `Crop`, `Shape`, `Text`, `Pen`, `Effects`, `Rotate`, `FlipHorizontal`, `FlipVertical`, `StrokeThickness`, `Circle`, `Arrow`, `Transform`, `Crop`, `free`, `original`, `square`, `3:1`, `3:2`, `4:3`, `5:4`, `16:9`, `Rotate`, `Flip`, `Reset`, `Undo`, `Redo`, `Save`, `Effects`, `Hue`, `Saturation`, `Brightness`, `Contrast`, `Blur` and `Sharpen`.
```

Comment:
`free` => 'custom'
mention in order:

HeaderItems, FooterItems, CropItems, ShapeItems, TextItems, PenItems, Effects

Reply:
Added a toolbar item names are in above mentioned order